app/controllers/wishlists_controller.rb

```diff
@@ -30,6 +30,25 @@ def destroy
     redirect_to wishlists_path, alert: t("generic_error")
   end
 
+  def edit
+    @wishlist = Wishlist.find(params[:id])
```

Comment:
this is exactly the same instruction like in other actions. You can extract it to `before_action`